.github/scripts/backport/main.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+GITHUB_TOKEN=$GITHUB_TOKEN
+
+git config --global user.email "noreply@github.com"
+git config --global user.name "GitHub Actions"
+
+git config --global credential.helper "store --file=~/.git-credentials"
```

Comment:
I don't believe you need to configure git credentials when on a github runner with the correct setup.